include/gatekeeper_gt.h

```diff
@@ -25,20 +25,27 @@
 #include <rte_tcp.h>
 #include <rte_udp.h>
 #include <rte_atomic.h>
+#include <rte_ip_frag.h>
 
 #include "gatekeeper_config.h"
 
 struct gt_packet_headers {
 	uint16_t outer_ip_ver;
 	uint16_t inner_ip_ver;
-	uint8_t l4_proto;
-	uint8_t priority;
-	uint8_t outer_ecn;
-
-	void *l2_hdr;
-	void *outer_l3_hdr;
-	void *inner_l3_hdr;
-	void *l4_hdr;
+	uint8_t  l4_proto;
+	uint8_t  priority;
+	uint8_t  outer_ecn;
+
+	void     *l2_hdr;
+	void     *outer_l3_hdr;
+	void     *inner_l3_hdr;
+	void     *l4_hdr;
+
+	/* Fields for parsing fragmented packets. */
+	bool     frag;
+	uint64_t l2_len;
+	uint64_t l3_len;
```

Comment:
To be clearer about what these variables are meant to held, maybe we could rename `info->l2_len` and `info->l3_len` to `info->l2_outer_l3_len` and `info->inner_l3_len` respectively?